src/components/SignUp/SignUp.vue

```diff
@@ -28,6 +28,15 @@
           </v-flex>
         </v-layout>
 
+        <v-text-field
+            data-cy="userName"
```

Comment:
This doesn't follow our coding convention of "firstword-nextword"
```suggestion
            data-cy="user-name"
```